src/StoreKeeper/WooCommerce/B2C/Tools/Media.php

```diff
@@ -132,6 +136,107 @@ public static function createAttachment($original_url)
         return get_post($attach_id)->ID;
     }
 
+    public static function createAttachmentAsCDN($cdnImageUrl, $bigImageUrl): ?int
```

Comment:
```suggestion
    public static function createAttachmentUsingCDN($cdnImageUrl): ?int
```

You should not need big url

Reply:
Removed big url.